docs/tr/docs/python-types.md

```diff
@@ -144,58 +140,94 @@ Yalnızca `str` değil, tüm standart Python tiplerinin bildirebilirsiniz.
+
+Dahili tipleri olan bu tiplere "**generic**"(jenerik) tipler denir. Ve bunları iç tipleriyle bile bildirmek mümkündür.
+
+Bu tipleri ve dahili tpileri bildirmek için standart Python küyüphanesi "typing"i kullanabilirsiniz. Bu tür tip belirteçlerini desteklemek için özel olarak mevcuttur.
+
+#### Python'ın Yeni Versiyonları İçin
+
+Yazım kullanan `sözdizimi`, Python 3.6'dan Python 3.9, Python 3.10 vb. dahil olmak üzere en son sürümlere kadar tüm sürümlerle **uyumludur**.
 
-"dict", "list", "set" ve "tuple" gibi diğer değerleri içerebilen bazı veri yapıları vardır. Ve dahili değerlerinin de tip belirtecleri olabilir.
+Python geliştikçe, **yeni sürümler** bu `belirteç` (tip eki) açıklamaları için gelişmiş destekle birlikte gelir ve çoğu durumda tür ek açıklamalarını bildirmek için yazım modülünü içe aktarmanıza ve kullanmanıza bile gerek kalmaz.
 
-Bu tipleri ve dahili tpileri bildirmek için standart Python modülünü "typing" kullanabilirsiniz.
+Projeniz için Python'un daha yeni bir sürümünü seçebilir ve böylece ekstra basitleştirilmiş özelliklerden yararlanabilirsiniz.
 
-Bu tür tip belirteçlerini desteklemek için özel olarak mevcuttur.
+Tüm dokümanlarda Python'un her sürümüyle uyumlu örnekler bulabilirsiniz (eğer bir farklılık oluştuysa).
 
-#### `List`
+Örneğin "**Python 3.6+**" Python 3.6 veya üzeri (3.7, 3.8, 3.9, 3.10, vb. dahil) ile uyumlu olduğu anlamına gelir. Ve "**Python 3.9+**" Python 3.9 veya üzeri (3.10 vb. dahil) ile uyumlu olduğu anlamına gelir.
+
+**Python'un en son sürümlerini** kullanayorsanız, en son sürüm için oluşturulan örnekleri kullanın. Bunlar **en iyi ve en basit sözdizimine** sahip olacaktır, örneğin, "**Python 3.10+**".
+
+#### `List` (Liste)
 
 Örneğin `str` değerlerden oluşan bir `list` tanımlayalım.
 
-From `typing`, import `List` (büyük harf olan `L` ile):
+=== "Python 3.9+"
 
-```Python hl_lines="1"
-{!../../../docs_src/python_types/tutorial006.py!}
-```
+    Declare the variable, with the same colon (`:`) syntax.
 
-Değişkenin tipini yine iki nokta üstüste (`:`) ile belirleyin.
+    As the type, put `list`.
 
-tip olarak `List` kullanın.
+    As the list is a type that contains some internal types, you put them in square brackets:
 
-Liste, bazı dahili tipleri içeren bir tür olduğundan, bunları köşeli parantez içine alırsınız:
+    ```Python hl_lines="1"
+    {!> ../../../docs_src/python_types/tutorial006_py39.py!}
+    ```
 
-```Python hl_lines="4"
-{!../../../docs_src/python_types/tutorial006.py!}
-```
+=== "Python 3.6+"
+
+    From `typing`, import `List` (büyük harf olan `L` ile):
+
+    ``` Python hl_lines="1"
+    {!> ../../../docs_src/python_types/tutorial006.py!}
+    ```
+
+    Değişkenin tipini yine iki nokta üstüste (`:`) ile belirleyin ve tip olarak `List` kullanın.
+
+    Liste, bazı dahili tipleri içeren bir tür olduğundan, bunları köşeli parantez içine alırsınız:
+
+    ```Python hl_lines="4"
+    {!> ../../../docs_src/python_types/tutorial006.py!}
 
 !!! ipucu
```

Comment:
`tip` için "ipucu" kullanmak daha mantıklı olacağı için `info` için `bilgi` kullanmak daha mantıklı olacaktır.